From 106341fef3b30476549a24dde6a9efcd50f07e06 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Tue, 18 Nov 2014 13:54:29 -0500
Subject: [redhat] kabi: Adding symbol find_next_zero_bit (lib/find_next_bit.c)

Message-id: <84be66aef2a114f653e13d49cdfd0e9936ab40bd.1416318767.git.poros@redhat.com>
Patchwork-id: 100143
O-Subject: [PATCH RHEL7.1 24/63] kabi: Adding symbol find_next_zero_bit (lib/find_next_bit.c)
Bugzilla: 1070274
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1070274
    https://bugzilla.redhat.com/show_bug.cgi?id=1070274

    Description:
    ============
    drbd uses bitmaps for tracking the differences between devices.
    Symbol used in drbd

Symbol History
==============
 Request:
 ========
 [PATCH RHEL7.0 47/80] kabi: Adding symbol find_next_zero_bit (lib/find_next_bit.c)
 Jiri Olsa <jolsa@redhat.com> (Tue,  4 Mar 2014 16:31:47 +0100)
 Description:
 ============
 drbd uses bitmaps for tracking the differences between devices

 Reviews:
 ========
 Re: [PATCH RHEL7.0 47/80] kabi: Adding symbol find_next_zero_bit (lib/find_next_bit.c)
 Prarit Bhargava <prarit@redhat.com> (Wed, 05 Mar 2014 07:54:32 -0500)
 Acked-by: Prarit Bhargava <prarit@redhat.com>
 P.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/find_next_zero_bit
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/find_next_zero_bit

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/find_next_zero_bit b/redhat/kabi/kabi-module/kabi_ppc64/find_next_zero_bit
new file mode 100644
index 0000000..e48ef79
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/find_next_zero_bit
@@ -0,0 +1,2 @@
+#1-
+0x479c3c86 find_next_zero_bit vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/find_next_zero_bit b/redhat/kabi/kabi-module/kabi_x86_64/find_next_zero_bit
new file mode 100644
index 0000000..e48ef79
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/find_next_zero_bit
@@ -0,0 +1,2 @@
+#1-
+0x479c3c86 find_next_zero_bit vmlinux EXPORT_SYMBOL
-- 
1.7.1